rust/definitions/src/qr_transfers.rs

```diff
@@ -1,31 +1,30 @@
-//! Information that could be send and received through air-gap as QR codes content  
+//! Information that could be send and received through air-gap as QR codes content
 //!
 //! All information that Signer could get without a complete wipe must arrive
 //! through the QR codes maintaining the air-gap. In addition to transactions
 //! that could be parsed and signed, this includes:
-//! - adding new networks,  
-//! - updating the metadata of existing networks,  
-//! - updating the types information,  
-//! - bulk-importing the derivations  
+//! - adding new networks,
+//! - updating the metadata of existing networks,
+//! - updating the types information,
+//! - bulk-importing the derivations
```

Comment:
It would be nice for reviewers not to make a huge patch even larger by including unrelated whitespace changes into it.